requirements.txt

```diff
@@ -29,6 +29,7 @@ setuptools==73.0.1
 sqlparse==0.5.1
 virtualenv==20.25.1
 wheel==0.44.0
+bleach>=6.0.0
 
```

Comment:
the python bleach library has been deprecated since the start of last year
can you find an alternative library to use?